bvm/ballerina-runtime/src/test/java/io/ballerina/runtime/test/cli/OperandTest.java

```diff
@@ -18,13 +18,193 @@
 
 package io.ballerina.runtime.test.cli;
 
+import io.ballerina.runtime.api.PredefinedTypes;
+import io.ballerina.runtime.api.creators.TypeCreator;
+import io.ballerina.runtime.api.values.BArray;
+import io.ballerina.runtime.api.values.BDecimal;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BString;
+import io.ballerina.runtime.internal.cli.CliSpec;
+import io.ballerina.runtime.internal.cli.Operand;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import java.math.BigDecimal;
+
 /**
  * Tests the options feature in bal cli commands.
  */
 public class OperandTest {
     @Test
-    public void testOperand() {
+    public void testString() {
+        Operand[] operands = {new Operand(false, "opString", PredefinedTypes.TYPE_STRING)};
```

Comment:
I think we can simply replace these with a data provider.

Reply:
The assertion part differs for each of the different types